DQMOffline/Muon/src/DiMuonHistograms.cc

```diff
@@ -111,18 +111,18 @@ void DiMuonHistograms::bookHistograms(DQMStore::IBooker& ibooker,
     TightTightMuon.push_back(ibooker.book1D("TightTightMuon" + EtaName[iEtaRegion],
                                             "InvMass_{Tight,Tight}" + EtaName[iEtaRegion],
                                             nBin[iEtaRegion],
-                                            LowMassMin,
-                                            LowMassMax));
+                                            HighMassMin,
+                                            HighMassMax));
     MediumMediumMuon.push_back(ibooker.book1D("MediumMediumMuon" + EtaName[iEtaRegion],
                                               "InvMass_{Medium,Medium}" + EtaName[iEtaRegion],
                                               nBin[iEtaRegion],
```

Comment:
These need to be `highMassMin`, `highMassMax`.

Reply:
I see, but this means changing their definition [here](https://github.com/cms-sw/cmssw/blob/master/DQMOffline/Muon/python/diMuonHistograms_cfi.py#L20-L23). The same would also apply to `LowMassMin` and `LowMassMax`, and this has nothing to do with the issue I would like to fix. Should I go ahead and fix the variables name as well?

Reply:
Yes, please fix this. Thanks!

Reply:
@llunerti , could you please follow up on this?